courses/offering/DataFormat.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Converts boolean values into emojis.
+ * @param {boolean} p - a boolean value that is either true or false.
+ * @returns {string} a string of emoji icon signifying true or false. If 'p' is neither true or false, returns an error message.
+ */
+function BooleanToEmoji(b) {
```

Comment:
Probably better implemented as
```js
return (b && "✅") || "❌"
```
or
```js
return b ? "✅" : "❌"
```

Reply:
> Thanks for contributing! Some non-critical comments have been added for your consideration. Otherwise, 🚀 🚀

Thanks for the suggestions! Appreciated 😄